### PR TITLE
Changes to fix the build on FreeBSD

### PR DESCRIPTION
**Pull request by bu7cher**
Add couple of fixes to be able build voipmonitor package on FreeBSD.
Tested on FreeBSD 13.0

### CODE REVIEW

File: Makefile.in [hunk 1]
```diff
@@ -73,10 +73,10 @@ LIBLD=@LIBLD@
 LIBLZMA=@LIBLZMA@
 LIBSSH=@LIBSSH@
 LIBGNUTLS=@LIBGNUTLS@
-LIBGNUTLSSTATIC=-lgcrypt -lgpg-error $(shell pkg-config gnutls --libs --static)
+LIBGNUTLSSTATIC= $(shell pkg-config gnutls --libs --static)
 SHARED_LIBS = ${LIBLD} -licuuc -licudata -lpthread -lpcap -lz -lvorbis -lvorbisenc -logg -lodbc ${MYSQLLIB} -lrt -lsnappy -lcurl -lssl -lcrypto ${JSONLIB} ${LIBSSH} -lxml2 -lrrd ${LIBGNUTLS} @LIBTCMALLOC@ ${GLIBLIB} ${LIBLZMA} -llzo2 ${LIBPNG} ${LIBFFT}
 STATIC_LIBS = -static @LIBTCMALLOC@ -licuuc -licudata -lodbc -lltdl ${LIBLD} -lrt -lz ${LIBSSH} -lcrypt -lm -lcurl -lssl -lcrypto -static-libstdc++ -static-libgcc -lpcap -lpthread ${MYSQLLIB} -lpthread -lz -lc -lvorbis -lvorbisenc -logg -lrt -lsnappy ${JSONLIB} -lrrd -lxml2 ${GLIBLIB} -lpcre -lz -ldbi -llzma ${LIBGNUTLSSTATIC} -llzo2 ${LIBPNG} ${LIBFFT} -lpthread ${SS7}
-INCLUDES = ${DPDKINC} -I/usr/local/include ${MYSQLINC} -I jitterbuffer/ ${JSONCFLAGS} ${GLIBCFLAGS}
+INCLUDES = ${DPDKINC} -I/usr/include ${MYSQLINC} -I jitterbuffer/ ${JSONCFLAGS} ${GLIBCFLAGS}
```
Comment (bu7cher) [inline Makefile.in]: This probably can break something on other than FreeBSD systems.

**Comment by milon21** [comment]
Hello,

we are looking into this.

**Comment by milon21** [comment]
I merged all commits without 'Fix compiler warnings' through my tree into our develop. 'Fix compiler warnings' commit needs more testing for our static building. Thanks.

**Comment by milon21** [comment]
Hello,

the patches are in the mainline. From 'Fix compiler warnings'  I got one thing only. I think you can cancel pull request now.

Many thanks and best regards.

**Pull request closed without merging**